com.github.TheThirdOne.rars.yml

```diff
@@ -0,0 +1,81 @@
+app-id: com.github.TheThirdOne.rars
+
+command: /app/bin/rars
+
+finish-args:
+  - --env=PATH=/app/jre/bin:/usr/bin
+  - --socket=x11
+  - --socket=fallback-x11
+  - --filesystem=xdg-desktop:rw
+  - --filesystem=xdg-documents:rw
+
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  # See https://github.com/flathub/org.freedesktop.Sdk.Extension.openjdk17#usage
+  - org.freedesktop.Sdk.Extension.openjdk17
+
+modules:
+  # See https://github.com/flathub/org.freedesktop.Sdk.Extension.openjdk17#usage
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk17/install.sh
+
+  - name: rars
+    buildsystem: simple
+    sources:
+      # See https://github.com/TheThirdOne/rars/releases
+      - type: file
+        url: https://github.com/TheThirdOne/rars/releases/download/v1.5/rars1_5.jar
+        dest-filename: rars.jar
+        sha512: 0644b78691167ba985724e7f545d44f7a0a64f2dc2b9e1b9544a7a4f56e58cbc7ba089bef4e0a7281528de3a5aa943b3be792782fa262211ec596a8104f0fabf
+      - type: script
+        commands:
+          # Get the screen width (See https://superuser.com/a/1567163)
+          - SCREEN_WIDTH=$(cat /sys/class/graphics/*/virtual_size | grep -Eow -m 1 "^([0-9]+)")
+
+          # Divide by 1920 (See https://stackoverflow.com/a/22406193)
+          - DEFAULT_SCALE=$(awk "BEGIN {printf \"%.3f\",${SCREEN_WIDTH} / 1920}")
+
+          # Get the scale
+          - RARS_SCALE="${RARS_SCALE:=$DEFAULT_SCALE}"
+
+          # Generate Java arguments
+          - RARS_JAVA="${RARS_JAVA:=-Dsun.java2d.uiScale=$RARS_SCALE}"
+
+          # Start
+          - /app/jre/bin/java ${RARS_JAVA} -jar /app/rars/rars.jar $@
+        dest-filename: rars
+      - type: file
+        path: com.github.TheThirdOne.rars.desktop
+      - type: file
+        path: com.github.TheThirdOne.rars.metainfo.xml
+      - type: file
+        path: icons/com.github.TheThirdOne.rars-16.png
+      - type: file
+        path: icons/com.github.TheThirdOne.rars-24.png
+      - type: file
+        path: icons/com.github.TheThirdOne.rars-32.png
+      - type: file
+        path: icons/com.github.TheThirdOne.rars-48.png
+      - type: file
+        path: icons/com.github.TheThirdOne.rars-64.png
+      - type: file
+        path: icons/com.github.TheThirdOne.rars-128.png
```

Comment:
128x128 is the only on required. You don't need to add antying else. A SVG is even better.